README.md

```diff
@@ -1,2 +1,45 @@
-# fns-monorepo
-Solidity Backend &amp; Frontend DApp Monorepo
+# Flare Naming Service Monorepo
+
+## Introduction
+
+Flare Name Service (FNS) is a decentralized domain name registry that facilitates the allocation of human-readable addresses to wallet addresses and smart contracts. FNS domains serve to simplify the identification of wallets and smart contracts on the Flare blockchain, as well as to enable links to decentralized webpages.
+
+Anyone with a Flare-supported wallet may visit the FNS dApp to register a .flr address of their choosing. This process effectively transforms their 64-character 0x address, which is typically composed of randomized alphanumeric characters, into an easily recognizable domain name.
```

Comment:
Consider adding a small section on: "The protocol also allows the setting of records of the users choosing, from arbitrary text inputs like email addresses, to wallet addresses for a multitude of other chains."

Reply:
resolved